be/src/exec/olap_meta_scanner.cpp

```diff
@@ -53,8 +55,34 @@ Status OlapMetaScanner::_init_meta_reader_params() {
     } else {
         _reader_params.tablet_schema = _tablet->tablet_schema();
     }
+    if (_parent->_meta_scan_node.__isset.column_access_paths) {
+        _reader_params.column_access_paths = &_parent->_column_access_paths;
```

Comment:
this implies that if the _meta_scan_node has the column_access_path, then the `_parent._column_access_path` must not be empty. 

Is the following code will be more readable?

```suggestion
    if (!_parent->_column_access_paths.empty()) {
        _reader_params.column_access_paths = &_parent->_column_access_paths;
```